opm/autodiff/MultisegmentWell_impl.hpp

```diff
@@ -638,7 +639,7 @@ namespace Opm
 
         // we need some intput the convergence calculations.
         // we don't need the exact values so we just use something ad-hoc
-        const std::vector<Scalar> B_avg = {800,1000,1};
+        const std::vector<Scalar> B_avg = {1000,800,1};
```

Comment:
I think the values of `B` are totally off. They should be below 1?

Reply:
Ops. It was meant has a quick fix for testing. I will use the competed values.